pages/agent/v2/cli_artifact.md.erb

```diff
@@ -226,7 +226,7 @@ Example:
    $ buildkite-agent artifact shasum "pkg/release.tar.gz" --build xxx
 
    This will search for all the files in the build with the path "pkg/release.tar.gz" and will
-   print to STDOUT it's SHA-1 checksum.
+   print to STDOUT its SHA-1 checksum.
```

Comment:
It might be worth rewording this one entirely. Maybe something like 

> This will search for all files in the build with the path "pkg/release.tar.gz" and will print the SHA-1 checksum of each one to STDOUT.